apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -238,7 +262,9 @@ module.exports = connect(function propsFromStore(state) {
     markdown: state.instructions.longInstructions,
     noVisualization: state.pageConstants.noVisualization,
     collapsed: state.instructions.collapsed,
-    documentationUrl: state.pageConstants.documentationUrl
+    documentationUrl: state.pageConstants.documentationUrl,
+    ttsInstructionsUrl: state.pageConstants.ttsInstructionsUrl,
```

Comment:
nit: it doesn't look like we're actually using the Instructions url, just the MarkdownInstructions